### DISCUSSION TITLE
Custom Inherited Process in Azure DevOps

### DISCUSSION THREAD

**Discussion by sinnitesh** [rfennell/AzurePipelines#1556] [General]
We are using custom Agile process to add some custom fields and I found that release notes not able to fetch related workitems and other relevant details.
Please let me know if this extension support it.

**Comment by rfennell** [comment]
Yes there is support for custom fields and related work items (assuming the relation is parent/child) in custom process templates. The details on using the extension is in the [wiki](https://github.com/rfennell/AzurePipelines/wiki/GenerateReleaseNotes---Node-based-Cross-Platform-Task).

But a couple of relevant points are:
- To access a custom field you need to use the lookup syntax e.g. `{{lookup this.fields 'MyProcess.MyField'}}`
- By default related work parent work items are not scanned for. To get parent and child work items of those directly associated with the build/release set `getAllParents` and/or `getParentsAndChildren` parameters
